entries/s/synology.com.json

```
@@ -7,7 +7,7 @@
       "totp",
       "u2f"
     ],
-    "documentation": "https://originwww.synology.com/en-us/knowledgebase/tutorials/615#t5",
+    "documentation": "https://kb.synology.com/en-us/DSM/tutorial/How_to_add_extra_security_to_your_Synology_NAS#t5",
```

```suggestion
    "documentation": "https://kb.synology.com/en-us/DSM/help/DSM/SecureSignIn/2factor_authentication",
```
This documentation page may be more useful.